h/accounts/schemas.py

```diff
@@ -203,13 +203,15 @@ class SocialLoginSignupSchema(colander.Schema):
     comms_opt_in = comms_opt_in_node()
 
 
-class EmailChangeSchema(CSRFSchema):
-    email = email_node(title=_("Email address"))
-    # No validators: all validation is done on the email field
-    password = password_node(title=_("Confirm password"), hide_until_form_active=True)
```

Comment:
Also removed `hide_until_form_active=True`, that was also used by the legacy form rendering.